### PR TITLE
MF-226 Allergy page is displayed blank

### PR DESCRIPTION
**Pull request by Gayatri-Kodag**
#MF-226 Allergy page is displayed blank

Updated the Environment Allergy Category to match Allergy Enum type

### CODE REVIEW

**Review by FlorianRappl — APPROVED** [review]
LGTM!

File: src/widgets/allergies/allergy-form.component.tsx [hunk 1]
```diff
@@ -299,6 +299,7 @@ export default function AllergyForm(props: AllergyFormProps) {
                   name="allergenType"
                   value={ENVIROMENTAL_ALLERGEN_CONCEPT}
                   onChange={handleAllergenChange}
+                  data-testid="environmentalAllergy"
                 />
                 <span id="ENVIROMENTAL">Enviromental</span>
```
Comment (FlorianRappl) [inline src/widgets/allergies/allergy-form.component.tsx:304]: Shouldn't that id be also `ENVIRONMENT`, just for consistency?

Comment (Gayatri-Kodag) [reply src/widgets/allergies/allergy-form.component.tsx:304]: Yes, I can change the test id to `ENVIRONMENT` for maintaining the consistency. In other tests, kebab case is used for test ids. Which one should I use as test id `ENVIRONMENT` or `environment-allergen-concept`?

Comment (FlorianRappl) [reply src/widgets/allergies/allergy-form.component.tsx:304]: If its really only about styling I would agree yes. Sounds good!

(I haven't checked where the ID / element is used)

**Pull request merged**